Snakefile

```diff
@@ -0,0 +1,16 @@
+import os
+import glob
+
+sql_paths = glob.glob("./data/*.sqlite")
+PLATE_IDS = [os.path.basename(fpath).rsplit(".", 1)[0] for fpath in sql_paths]
```

Comment:
FYI, I have trended away from `os` in favor of [`pathlib`](https://docs.python.org/3/library/pathlib.html)

I have found it to be more intuitive and functionally complete.

Reply:
replaced `os` with `pathlib` 

commit: d74575671a20747cfe0ac21744b162584cc915de